packages/zwave-js/src/lib/commandclass/BarrierOperatorCC.ts

```diff
@@ -81,22 +81,22 @@ export class BarrierOperatorCCSet extends BarrierOperatorCC {
 				ZWaveErrorCodes.Deserialization_NotImplemented,
 			);
 		} else {
-			this.targetValue = options.targetValue;
+			this.state = options.state;
 		}
 	}
 
-	public targetValue: boolean;
+	public state: State.Open | State.Closed;
 
 	public serialize(): Buffer {
-		const payload: number[] = [this.targetValue ? 0xff : 0x00];
+		const payload: number[] = [this.state ? 0xff : 0x00];
 		this.payload = Buffer.from(payload);
 		return super.serialize();
 	}
 
 	public toLogEntry(): MessageOrCCLogEntry {
 		return {
 			...super.toLogEntry(),
-			message: { "target value": this.targetValue },
+			message: { "state value": this.state },
```

Comment:
`"state value"` sounds weird.

Reply:
`"target value"` or `"target state"` ?  
I prefer the later

Reply:
target state 👍🏻 
(or just state)